auto_submit/lib/service/validation_service.dart

```diff
@@ -334,25 +341,22 @@ Exception: ${exception.message}
     final int number = messagePullRequest.number!;
 
     try {
-      // The createGitHubGraphQLClient can throw Exception on github permissions
-      // errors.
-      final graphql.GraphQLClient client = await config.createGitHubGraphQLClient(slug);
-      graphql.QueryResult? result;
+      github.PullRequestMerge? result;
 
       await retryOptions.retry(
         () async {
           result = await _processMergeInternal(
-            client: client,
-            config: config,
-            queryResult: queryResult,
-            messagePullRequest: messagePullRequest,
+            githubService,
+            slug,
+            number,
+            github.MergeMethod.squash,
```

Comment:
Can we leave a TODO to make this configurable by repo? For Flutter, we use squash, but as we roll out to other repos they may want to configure this.

Reply:
So I have been thinking about configuration. Do you mean configuration outside of config.dart?

Reply:
Yes. This should be configured via a repo or orgs .github directory.

See https://www.freecodecamp.org/news/how-to-use-the-dot-github-repository/

Reply:
Added a TODO. Will make that change in a different PR.